modules/feedbackProducer/src/idl.thrift

```diff
@@ -29,64 +29,64 @@ service feedbackProducer_IDL
     * @param template_tag_ is the tag of the template.
     * @return true/false on success/failure.
     */
-   bool setTemplateTag(1:string template_tag_);
+   bool setTemplateTag(1:string template_tag);
```

Comment:
Good that you removed the trailing underscore, but have you updated the C++ code accordingly?

Reply:
Done.